getid3/extension.cache.dbm.php

```diff
@@ -179,7 +207,7 @@ public function clear_cache() {
 
 
 	// public: analyze file
-	public function analyze($filename) {
+	public function analyze($filename, $filesize=null, $original_filename='') {
```

Comment:
Declaration of getID3_cached_dbm::analyze($filename) should be compatible with getID3::analyze($filename, $filesize = NULL, $original_filename = '')